### PR TITLE
Add documentation about provisioning

### PR DESCRIPTION
**Pull request by nodanaonlyzuul**
Have time to look this over @holingpoon?
It's by no means exhaustive but it's a good start.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -74,11 +74,56 @@ Our branches (in order or stability are):
 
 We use Travis for continuous deployment.
 Merging to certain branches automatically deploys to the environment associated to
-that branch.
+that branch. Here are the [travis build settings](https://travis-ci.org/NYPL-discovery/scsb_item_updater/settings).
 
 Merging `master` => `development` automatically deploys to the development environment. (after tests pass)
```
Comment (kfriedman) [inline README.md]: Just a heads-up, this is a different Git workflow than is being practiced by other members of the team:

https://github.com/NYPL/staff-picks#git-workflow

We should probably have a consistent workflow; especially as we continue down the exciting road of CD. I think someone should open up an issue in engineering-general to create a standard.

Comment (nodanaonlyzuul) [reply README.md]: Yeah this is something that should be resolved because this _is_ the workflow for still other apps.

Comment (kfriedman) [reply README.md]: Yes, I know. This is the workflow most familiar to me too. I think this is a result of having previously separate teams. Would you be willing to open an issue in engineering-general? If not, I can try to find some time to coordinate. 😬

**Comment by nodanaonlyzuul** [comment]
Hey - @nonword/ @holingpoon

I'm doing some open PR spring (Summer?)( cleaning.
I've totally lost context about what this is.
Feel free to close it though if it no longer makes sense.

**Pull request closed without merging**